bats_test/downloads/file-downloads/publlic_files.bats

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bats
+
+# Support lib
+load '/home/node/packages/node_modules/bats-support/load.bash'
+# File methods. DEPENDS on SUPPORT lib.
+load '/home/node/packages/node_modules/bats-file/load.bash'
+# Assertions.  DEPENDS on SUPPORT lib.
+load '/home/node/packages/node_modules/bats-assert/load.bash'
+
+DIR="$(cd "$(dirname "$BATS_TEST_FILENAME")" >/dev/null 2>&1 && pwd)"
+
+setup_file() {
+    run $DIR/pre_download.sh
+    echo "##### Downloaded 7 files" >>/home/node/bats.log
```

Comment:
What log should we dump here? Custom log messages?

Reply:
This is how you create log files according to BATS docs, yes